src/main/java/frc/robot/RobotContainer.java

```diff
@@ -106,8 +101,17 @@ private void controllerBindings() {
     new JoystickButton(controlXbox, 3).whenPressed(new ElevatorStopCommand(elevatorSubsystem));
 
     Runnable tank = () -> {
-      tankSubsystem.setDrivePowers(-controlXbox.getY(Hand.kLeft), controlXbox.getX(Hand.kRight));
 
+      // Check which controller is being used
+      boolean isXbox = (Math.abs(controlXbox.getY(Hand.kLeft))
+          + controlXbox.getX(Hand.kRight)) > (Math.abs(joystickLeft.getY()) + Math.abs(joystickRight.getY()));

```

Comment:
As noted elsewhere, this is missing an `abs()`

Reply:
This will take the strongest input as for the control, which may not be what we want. It may be better to compare against 0 or epsilon from zero to determine if an input is active. How would you resolve if both are active at the same time? You could default to `isXbox = false` or you could have some way to hold on to whatever was active previously.